tests/tests.js

```diff
@@ -637,19 +675,26 @@ QUnit.test('processTree: head tag', function(assert) {
 QUnit.test('minimizeStyles: root html tag', function(assert) {
   var message = {
     'html': [
-        '<html',
+        '<html id="myId" ',
         'style="animation-delay: 0s; width: 5px;" ',
         '></html>'
     ],
     'frameHoles': null,
     'idToStyleIndex': {},
+    'idToStyleMap': {
+      'myId': {
+        'animation-delay': '0s',
+        'width': '5px'
+      }
+    },
     'windowHeight': 5,
     'windowWidth': 5,
+    'rootId': 'myId',
     'rootStyleIndex': 1,
     'frameIndex': '0'
   };
   minimizeStyles(message);
-  assert.equal(message.html[1], 'style="width: 5px;" ');
+  assert.equal(message.html[1], 'style="width:5px;" ');
```

Comment:
You said you found an issue where the old regex approach didn't work but the new approach does. Can you add a test of that case?